docs/source/intro_to_nwb/file_structure.rst

```diff
@@ -1,2 +1,160 @@
+      │   │   │       ├── data
+      │   │   │       ├── reference_frame
+      │   │   │       └── timestamps
+      │   │   └── licks
+      │   │       └── timestamps
+      │   └── ecephys
+      │       └── LFP
+      │           └── LFP
+      │               ├── data
+      │               ├── electrodes
+      │               └── starting_time
+      ├── session_description
+      └── session_start_time
+
+At a glance, you can see that there are datasets within the file that represent
+acquired (raw) electrical data, processed position data, and processed LFP data.
+There are also metadata fields that represent the institution and lab where the data was
+collected, a description of the session, and the session start time.
+
+An NWB file often consists of many different data and metadata so as to provide a complete
+representation of the data that was collected in a session. This may seem overwhelming
+at first, but NWB provides tools to make it easy for you to read and write data in the NWB format,
+as well as inspect and visualize your NWB data. Community tools leverage the standardized
+structure of an NWB file to find relevant data and metadata for data processing, analysis,
+and visualization. And once you understand the structure of an NWB file, you can open any
+NWB file and quickly make sense of its contents.
+
+File hierarchy
+--------------
+
+An NWB file consists of four basic types:
+
+* groups, which are like folders or directories in a filesystem and can contain any of the basic types
+* datasets, which are N-dimensional arrays
+* attributes, which are small metadata values attached to a group or dataset, like folder/file properties in a
+  filesystem
+* links, which point to a group or dataset, like shortcuts or aliases in a filesystem
+
+TODO: add image
+
+Raw data
+--------
+
+Raw data are stored in NWB in containers placed in the ``acquisition`` group
+at the root of the NWB file.
+In PyNWB, you can add raw data to an NWB file using NWBFile.add_acquisition.
+In MatNWB, you can add raw data to an NWB file using
+
+See [TODO link] for more details.
+
+Processed data
+--------------
+
+Processed data, for example, the extracted fluorescence time series
+from ROIs in calcium imaging data, are stored in NWB in containers
+within a ``ProcessingModule`` container placed in the ``processing`` group
+at the root of the NWB file.
+Any scripts or software that process raw data and generate processed
+data should store the results within a ``ProcessingModule``.
+
+The name of the ``ProcessingModule`` reflects the type of processed data
+within it.
+
+.. list-table::
+    :header-rows: 1
+
+    * - Type of processed data
+      - Name of ``ProcessingModule``
+    * - extracellular electrophysiology
+      - "ecephys"
+    * - intracellular electrophysiology
+      - "icephys"
+    * - optical physiology
+      - "ophys"
+    * - behavior
+      - "behavior"
+
+Containers
+------------------
+TODO
+
+TimeSeries
+------------------
+)
```

Comment:
Mention concept of TimeIntervals table for annotation, e.g., trials

Move timestamp and units info here. Remove subheaders of those.